### PR TITLE
Add mention of the WHATNOT meeting

### PR DESCRIPTION
**Pull request by cwilso**

### CODE REVIEW

File: Working Mode.md [hunk 1]
```diff
@@ -147,6 +147,8 @@ Implementation disagreement should not result in implementation-defined behavior
 
 Most Workstreams don't run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liaise with other groups.
 
+There is a regular biweekly triage meeting to unblock issues across all WHATWG Workstreams.  Topics may come from any WHATWG repository; you can find the information on the next meeting by looking for the [Upcoming WHATNOT meeting](https://github.com/whatwg/html/issues?q=is%3Aissue+is%3Aopen+label%3Aagenda%2B+Upcoming) in the HTML repository.  Topics are added to the agenda for the next meeting by adding the "agenda+" label in any WHATWG repository.  The timing of this meeting changes to distribute across time zones.
```
Comment (zcorpan) [inline Working Mode.md:150]: Please avoid biweekly since it means both every two weeks and twice a week.

Comment (cwilso) [reply Working Mode.md]: Noted and corrected.

File: Working Mode.md [hunk 2]
```diff
@@ -147,6 +147,8 @@ Implementation disagreement should not result in implementation-defined behavior
 
 Most Workstreams don't run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liaise with other groups.
 
+There is a regular triage meeting held approximately every other week to unblock issues across all WHATWG Workstreams.  Topics may come from any WHATWG repository; you can find the information on the next meeting by looking for the [Upcoming WHATNOT meeting](https://github.com/whatwg/html/issues?q=is%3Aissue+is%3Aopen+label%3Aagenda%2B+Upcoming) in the HTML repository.  Topics are added to the agenda for the next meeting by adding the "agenda+" label in any WHATWG repository.  The timing of this meeting changes to distribute across time zones.
```
Comment (annevk) [inline Working Mode.md:150]: Nits:

* Single space after periods.
* Either HTML standard repository or `html` repository, prolly the former.
* "in any WHATWG standard repository" (we don't add labels to all repositories)

**Review by annevk — COMMENTED** [review]
Nice!

File: Working Mode.md (hunk 2 above)
Comment (cwilso) [reply Working Mode.md]: All fixed.  (Sorry, I have an involuntary two-space-after-period tic, of course I intended to match style.  :) )

File: Working Mode.md [hunk 3]
```diff
@@ -147,6 +147,8 @@ Implementation disagreement should not result in implementation-defined behavior
 
 Most Workstreams don't run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liaise with other groups.
 
+There is a regular triage meeting held approximately every other week to unblock issues across all WHATWG Workstreams. Topics may come from any WHATWG repository; you can find the information on the next meeting by looking for the [Upcoming WHATNOT meeting](https://github.com/whatwg/html/issues?q=is%3Aissue+is%3Aopen+label%3Aagenda%2B+Upcoming) in the HTML standard repository. Topics are added to the agenda for the next meeting by adding the "agenda+" label in any WHATWG standard repository. The timing of this meeting changes to distribute across time zones.  We may also hold similar triage meetings in-person alongside other industry events; we did so in 2023 as the [WHATUP meeting](https://github.com/whatwg/meta/issues/284).
```
Comment (annevk) [inline Working Mode.md:150]:
```suggestion
There is a regular triage meeting held approximately every other week to unblock issues across all WHATWG Workstreams. Topics may come from any WHATWG repository; you can find the information on the next meeting by looking for the [upcoming WHATNOT meeting](https://github.com/whatwg/html/issues?q=is%3Aissue+is%3Aopen+label%3Aagenda%2B+Upcoming) in the HTML standard repository. Topics are added to the agenda for the next meeting by adding the "agenda+" label in any WHATWG standard repository. The timing of this meeting changes to distribute across time zones. We may also hold similar triage meetings in-person alongside other industry events; we did so in 2023 as the [WHATUP meeting](https://github.com/whatwg/meta/issues/284).
```

**Review by annevk — APPROVED** [review]

**Comment by cwilso** [comment]
Thanks Anne!

**Pull request merged**